docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md

```diff
@@ -10,6 +10,8 @@ Desktop Modeler can directly deploy diagrams and start process instances in Camu
 
 ![deployment icon](./img/deploy-icon.png)
 
+<!--- These screenshots reference 8.2, can we update these? --->
```

Comment:
@nikku See the only comment I have for Modeler here 👍

Reply:
Modeler issue: https://github.com/camunda/camunda-modeler/issues/4048.